src/scripts/farming/Plot.ts

```diff
@@ -143,6 +147,10 @@ class Plot implements Saveable {
                 auraStr.push(`Death: ×${this.auraDeath().toFixed(2)}`);
             }
 
+            if (this.auraDecay() !== 1) {
+                auraStr.push(`Decay: ${this.auraDecay().toFixed(2)}x`);
```

Comment:
`x` was moved in front of the numbers after you wrote this
```suggestion
                auraStr.push(`Decay: ×${this.auraDecay().toFixed(2)}`);
```